package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "bDev",
```

Comment:
I read [the docs](https://docs.npmjs.com/getting-started/using-a-package.json) this time. Lets use our git repo name: blockly-devtools